VacationCalendar/VacationCalendar.BusinessLogic/Models/VacationRequest.cs

```diff
@@ -6,15 +6,95 @@
 
 namespace VacationCalendar.BusinessLogic.Models
 {
-    internal class VacationRequest
+    public class VacationRequest
     {
         public int Id { get; set; }
         public DateTime From { get; set; }
         public DateTime To { get; set; }
 
-        public int  Days 
-        { 
-            set { } 
+        public TimeSpan NumberOfDays 
+        {
+            get 
+            {  
+                return To.Subtract(From); 
+            }
+        }
+        /// <summary>
+        /// Metoda oblicza dni wakacji, pomija soboty i niedziele
+        /// </summary>
+        /// <param name="from"></param>
+        /// <param name="to"></param>
+        /// <returns></returns>
+        public int CountVacationDays(string from, string to, out string message)
```

Comment:
Jeśli faktycznie wrzuciłeś tutaj metodę do tej klasy to chyba nie powinna ona już przyjmować tylu parametrów. 
From i Tu powinno zostać zabrane z modelu i tutaj wtedy masz ładną enkapsulację. Co do samego sposobu pisania kodu to można tak jak mówić wywalić te metody do serwisu i się nie przejmować, zostawić tutaj tzw "Anemic model". 
Natomiast takie bardziej nowoczesne podejście faktycznie pozwala na dodawanie logiki do takiej encji tak jak zaproponował Kuba. Możemy iść w tę stronę i zobaczymy co nam z tego wyjdzie. Natomiast jeśli chcesz metodę która przyjmuje parametry wgl nie z tego modelu i chcesz miec te **from i to** w parametrach to  zastanowiłbym się czy nie warto tego wrzucić do osobnej klasy i mieć to nawet na ten moment jako metoda statyczna wtedy.